llama_toolchain/inference/adapters/tgi/config.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Optional
+
+from huggingface_hub import HfApi
```

Comment:
we want the config files to be extremely lightweight so one can use them outside of the larger containers. as such they should only have datatypes and no "behavior" or functions at all. I suggest you move at least the `get_namespace()` method out into the impl where it needs to be used so this dependency is not here.

Reply:
Fixed ✅